spec/models/user_spec.rb

```diff
@@ -0,0 +1,46 @@
+require 'rails_helper'
+require 'faker'
+
+describe User, type: :model do
+  it do
+    should validate_presence_of(:first_name)
+  end
+  it do
+    should validate_presence_of(:last_name)
+  end
+
+  subject(:user) do
```

Comment:
same here, you can move all the user creation to the factory file

Reply:
Done